src/models/Settings.php

```diff
@@ -57,6 +57,13 @@ class Settings extends Model
      */
     public $keyPrefix = '';
 
+    /**
+     * Max kilobytes of the X-Cachetag header
+     *
+     * @var string
```

Comment:
should be `int` not `string`

Reply:
Fixed in https://github.com/ostark/upper/pull/57/commits/b493328e9164b9c23ebb7165908d86dc0cb25dc9

Reply:
I guess I was thinking that if this came from the ENV it’d be a string. But that’s silly, on the setting mode we have access to real types so let’s use em’! Good catch.